### DISCUSSION TITLE
Introduce Yourselves

### DISCUSSION THREAD

**Discussion by Montoya** [MetaMask/snaps#343] [General]
Welcome to the MetaMask Snaps Discussions Board for builders! If you are building a snap or thinking about building a Snap, feel free to introduce yourselves here. Also, please fill out our [Snaps Builders Onboarding Form](https://docs.google.com/forms/d/e/1FAIpQLSdAtVvHcl9xhIdJdTDSd1YXGN1GdLWzMfAFhNTqJElX90iHOw/viewform?usp=sf_link) so we can stay in touch!

**Comment by shawnmitchell** [comment]
👋 I'm shawnmitchell.eth and I'm working on a distributed, censorship-resistant, self-sovereign identity product now called IDPass.eth - formerly RealID.eth - and developing a Snap to allow an IDPass holder to securely decrypt and build Merkle proofs for discrete, verifiable personal information within the security of the wallet environment.

**Comment by ghost** [comment]
Hi, I'm Orange, I'm working on snaplist.org , just a fork from Chainlist, which make user easily to install a snap.

**Comment by piotr-roslaniec** [comment]
Hello. My name is Piotr and I'm building snaps and snap-related utilities:
https://github.com/piotr-roslaniec/awesome-metamask-snaps

**Comment by erfan377** [comment]
Hi!
Erfan Here! Have been working on known address / contract management, phishing prevention bullet point. I'm exploring snap integration. Would definitely appreciate metamask team's feedback while building it.

**Comment by HarishRocks** [comment]
Hello everyone,
I'm a fullstack developer from [Cypherock](https://cypherock.com/). We make the safest hardware wallets by never storing the private keys at a single place. 
I'm very excited to be a part of the Metamask snap ecosystem. I'll be exploring snap to build an integration to our hardware wallet.

**Reply by Montoya** [reply]
Welcome! Do you need USB connectivity for this? If so, we don't have a USB connection permission yet, but we are planning to add something soon. If QR code is enough then you might be able to proceed already with that approach.

**Reply by HarishRocks** [reply]
Hey @Montoya , we do need USB connectivity for communication 😬 
When can we expect it to be included as a part of snaps project?
In the event it gets delayed, what other alternatives can we explore?

**Comment by cryptoKevinL** [comment]
Hey Everyone,

I am co-founder of [WalletChat](https://walletchat.fun) which is already built as a chrome extension and web app, providing wallet-to-wallet chat and social experiences.  Looking forward to help shape MM Snaps and expand its uses.  

I also do smart contract security reviews and have some ideas for proof of ownership/signer only wallets which I think would work well with MM Snaps.

**Comment by zengbing15** [comment]
Hi Everyone,
I am Bella, a Developer Advocate. I'm working on the connection between Wallet and DApps and now I'm researching the metamask-snaps.

**Comment by andyv09** [comment]
Hello everyone,

I'm Andy. I'm working as a researcher & developer @ [BlockchainLab:UM](https://blockchain-lab.um.si/?lang=en) and am currently working on [SSI Snap](https://github.com/blockchain-lab-um/ssi-snap).

Nice to meet y'all!

**Comment by andreolf** [comment]
Hello everyone, I am Francesco Andreoli, part of DevRel team and supporting snaps developers. Exited for this journey!

**Comment by LidamaoHub** [comment]
Hi everyone,I am Dave,a fullstack Developer from [DappReader](DappReader.com),I'm developing a metamask-snap that helps user to audit smart contracts easily with AI before the transaction confirmed.Hoping to do more cool stuff in the future

**Comment by shubham-yug** [comment]
Hello everyone, I am Shubham, part of [Yug Network](https://yug.network/), developing a metamask-snap that helps users to ensure that they don’t transact with any sanctioned individual thereby curbing illicit financing and money laundering. 

[yug-snap](https://github.com/yug-network/yug-snap.git)

Thanks

**Comment by NasCorp** [comment]
Hey, my name is Nik, and I'm working on a brand-new snap. Now I'm searching for a way to solve the problem:
```
Cannot install version "0.1.0" of snap "npm:@nascorp/chain-track-snap": The snap is not on the allow list.
```
Can you help me?

**Reply by Montoya** [reply]
Hi, you should be using MetaMask Flask to test Snaps you are building. https://metamask.io/flask/

If you would like your Snap to be allowlisted for the extension, then there is a process for that described here: https://github.com/MetaMask/snaps/discussions/1411

**Comment by gosunuts** [comment]
Hello, my name is Kim, backend engineer on [aergo](https://github.com/aergoio/aergo). I`m working on new wallet using snaps.
nice to meet you!

**Reply by TravCrypto** [reply]
Hey @gokch and welcome! 👋

**Comment by daglak** [comment]
hello my name is JOHRI ISMAIL

nice to meet you everyone

**Comment by Bravark** [comment]
Hi, i am Arowoshola Zaccheus from [Pouchpay](pouchpay.xyz) we want to build an Interoperability snap for the Tron blockchain, so users of metamask can use trx and trc20 token inside metamask.

**Reply by Montoya** [reply]
That's pretty cool! Welcome!

**Comment by Morgosh** [comment]
[45130a49-b3cd-4f5f-8f1a-0d8d5b413f9d.webm](https://github.com/user-attachments/assets/e2b98ea8-e12f-4bb6-8ef7-4439d3fe68fa)

Hello everyone! I just deployed the first version of our metamask snap for handling ZK native transactions on ZKEVM and is available on metamask flask https://zk-snap-site.vercel.app/.

I am also attaching a video where I explain what exactly the snap does. Also I explain the difficulties we're facing due to metamask snap limitations which ideally we would like solved as we believe this snap can prevent vulnerabilities ZK native transactions have. 

Looking forward to your feedback and suggestions!

**Reply by ziad-saab** [reply]
Welcome to Snaps and thank you for sharing your feedback! Hit me up on telegram @ziadmtl and I'll invite you to the Snaps Developers channel so you can stay up to date on new features and fixes. We do plan to make signature insights available before pressing the "Sign" button once the signature confirmation screen allows this.
